src/utils/index.ts

```diff
@@ -49,3 +49,12 @@ export const randomMinutes = (min: number, max: number) => {
 export const randomSeconds = (min: number, max: number) => {
   return 1000 * between({ min, max });
 };
+
+export const getDayOfYear = (date: Date) => {
+  const now = new Date(date);
```

Comment:
If you have to pass in a type of Date, is line 54 needed? I could see it if you could pass in a type of Date, or string, or number. Unless I'm missing something.